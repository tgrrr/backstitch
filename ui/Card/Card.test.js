import React from 'react'
import renderer from 'react-test-renderer'
import { Card } from '.'

it('Card: default', () => {
    const component = renderer.create(
        <Card>
            <>
                Card
            </>
        </Card>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
