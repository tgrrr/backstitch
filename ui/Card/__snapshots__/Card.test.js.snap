// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Card "
    data-testid="Card"
  >
    <div
      className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
    >
      Card
    </div>
  </div>
</div>
`;
